### DISCUSSION TITLE
Sprite texture not rendering in the correct position.

### DISCUSSION THREAD

**Discussion by Kitanga** [pixijs/pixijs#7340] [Q&A] [answered]
### Expected Behavior
The sprite's texture should be where the sprite is

### Current Behavior
![Image of bug](https://aws1.discourse-cdn.com/free1/uploads/phaser1/original/2X/d/d26bc6c08607ee97cff571c94b49af8ba5c93683.png)

PIXI explorer extension shows that this is where the sprite should be but when I move my mouse to the position of the sprite nothing happens (I placed listeners on the sprites I made).

### Possible Solution

No idea why this is happening, help urgently needed.

### Steps to Reproduce
Go here https://codepen.io/kitanga/pen/BapjgZZ?editors=1010

Scroll until you are on the center, I removed the css that kept it on screen just to make sure that might not be what's causing the issue.

### Environment

- **`pixi.js` version**: _e.g. 6.0.1_
- **Browser & Version**: _e.g. Chrome 89_
- **OS & Version**: _e.g. Windows 10_
- **Running Example**: _https://codepen.io/kitanga/pen/BapjgZZ?editors=1010_

**Comment by bigtimebuddy** [comment, marked as answer]
Try this: https://codepen.io/bigtimebuddy/pen/MWJKNbg?editors=1010

* pointerover/pointerout instead of pointerenter/pointerleave
* use smaller canvas size
* temporarily removed the RenderTexture caching of the shape

This is a HUGE canvas, things start to be unstable at such a large sizes. If you're building a map, which it looks like you are consider something like https://github.com/davidfig/pixi-viewport.

**Reply by Kitanga** [reply]
Oh, sorry for the delay lol, was continuing with my tests waiting for any solution.

**Reply by Kitanga** [reply]
I'm a big fan of pixi-viewport, definitely we'll bring this up to the guys at work. But for now let me check the code pen you shared. Thanks for the assistance btw.

**Reply by Kitanga** [reply]
That's it, thanks, so the size was an issue, and I thought setting the resolution would do what, I believe, autoDensity prop does. interesting.
